thunder/extend/__init__.py

```diff
@@ -181,12 +187,19 @@ def _meta(*args):
 
         def _bind_postprocess(bsym: BoundSymbol) -> None:
             bsym.subsymbols = tuple(bsyms)
-            bsym._call_ctx: dict[str, Callable] = {name: fn}
+            bsym._call_ctx = {name: fn}
 
         sym = Symbol(name=name, meta=_meta, is_fusion=True, _bind_postprocess=_bind_postprocess, executor=self)
         return sym.bind(*inputs, output=outputs)
 
 
+@run_once
+def warn_default_meta(opname: str, exname: Hashable):
+    warnings.warn(
+        f"Meta function has not been specified for operator {opname} in exexutor {exname}. Defaulting to running the operator with example tensors to determine its output metadata."
```

Comment:
"exexutor"

Reply:
Removed the whole function, ripped out default meta, we can talk about it later.